tests/app/main/views/uploads/test_upload_contact_list.py

```diff
@@ -208,7 +208,7 @@ def test_upload_csv_file_shows_error_banner(
     mock_upload.assert_called_once_with(
         filedata='',
         region='us-west-2',
-        bucket_name='test-contact-list',
+        bucket_name='local-contact-list',
```

Comment:
So, it looks like the "test" buckets in the app are actually the "dev" buckets? I don't know that we should do anything now, but it would be nice to test these in a more isolated way.

Reply:
Yeah, I think they're all mocked out, so I'll see if I can easily change the test names to verify that. The old test bucket names didn't actually exist